bitmap-transformer/src/main/java/Bitmap.java

```diff
@@ -0,0 +1,44 @@
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class Bitmap {
+
+    //Instance Variables
+    public Path path;
+    public BufferedImage imageData;
+
+    //Constructor
+    public Bitmap(Path imagePath){
+        this.path = imagePath;
+        BufferedImage img = null;
+        try {
+            img = ImageIO.read(imagePath.toFile());
+            this.imageData = img;
+        } catch (IOException e) {
+
+        }
+    }
+
+    //Instance Methods
+    public void flipHorizontally(){
+        for int i = 0; i < this.imageData.getHeight(); i++){
+            for int j = 0; j < this.imageData.getWidth() / 2; j++){

```

Comment:
take another look at these two for loops - you're missing something at the start of each loop so that the method can't run.....